frontend/src/components/ListViews/TasksListView.vue

```diff
@@ -41,7 +41,9 @@
         <div v-if="column.key === 'due_date'">
           <Tooltip :text="dateFormat(item, 'ddd, MMM D, YYYY | hh:mm a')">
```

Comment:
```suggestion
          <Tooltip
            :text="item && dateFormat(item, 'ddd, MMM D, YYYY | hh:mm a')"
          >
```

Only show tooltip if date is set